concrete-core/src/specification/engines/lwe_ciphertext_vector_loading.rs

```diff
@@ -0,0 +1,47 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::LweCiphertextVectorEntity;
+use concrete_commons::parameters::LweCiphertextRange;
+
+engine_error! {
+    LweCiphertextVectorLoadingError for LweCiphertextVectorLoadingEngine@
+    DisorderedInputRange => "The input range bounds must be ordered.",
```

Comment:
As before, I do not understand the error DisorderedInputRange => "The input range bounds must be ordered.",

Reply:
Thanks for the remarks ! 

@tmontaigu: I also think Unorder is better. I changed it !

@ilaria-zama: Indeed, this error is tricky to understand... Basically this function will load a sub part of the ciphertext vector into a newly created ciphertext vector. To specify the range we want to load, we pass the `range` argument, which is of type `LweCiphertextRange`. If you navigate to its definition you will see that it is nothing more than a struct holding two bounds:
```rust
pub struct LweCiphertextRange(pub usize, pub usize);
```
Implicitly, we expect the first usize to be smaller or equal to the second, but we can not verify that at compile time :( 
Since there is no particularly better place to verify this order, I chose to put it in the engine trait itself, hence the error variant.

Does that make sense ?

Reply:
Discussing with @ilaria-zama and @jborfila , we decided to leave this that way, before a more integrated way to create parameters can be designed.